packages/expo-module-template/src/{%- project.name %}Module.web.ts

```diff
@@ -0,0 +1,15 @@
+import { EventEmitter } from 'expo-modules-core';
+
+const emitter = new EventEmitter({} as any);
+
+export default {
+  PI: Math.PI,
+  async setValueAsync(value: string): Promise<void> {
+    emitter.emit('onChange', { value });
+  },
+  hello() {
+    return 'Hello world! 👋';
+  },
+  startObserving() {},
+  stopObserving() {},
```

Comment:
Is this necessary? 🤔

Reply:
Nope, but now I came up with an idea that we can add these comments [from this docs page](https://reactnative.dev/docs/native-modules-ios#sending-events-to-javascript):
```suggestion
  startObserving() {
    // Optional, this will be called when this module's first listener is added.
  },
  stopObserving() {
    // Optional, this will be called when this module's last listener is removed.
  },
```